mlflow/langchain/databricks_dependencies.py

```diff
@@ -0,0 +1,185 @@
+from typing import Dict, Set
+
+_DATABRICKS_DEPENDENCY_KEY = "databricks_dependency"
+_DATABRICKS_VECTOR_SEARCH_INDEX_NAME_KEY = "databricks_vector_search_index_name"
+_DATABRICKS_VECTOR_SEARCH_ENDPOINT_NAME_KEY = "databricks_vector_search_endpoint_name"
+_DATABRICKS_EMBEDDINGS_ENDPOINT_NAME_KEY = "databricks_embeddings_endpoint_name"
+_DATABRICKS_LLM_ENDPOINT_NAME_KEY = "databricks_llm_endpoint_name"
+_DATABRICKS_CHAT_ENDPOINT_NAME_KEY = "databricks_chat_endpoint_name"
+
+
+def _assign_value_or_append_to_list(d, key, value):
+    if key in d:
+        if isinstance(d[key], list):
+            d[key].append(value)
+        else:
+            d[key] = [d[key], value]
+    else:
+        d[key] = value
```

Comment:
Would it be simpler and less confusing to cast them all to lists instead of preserving scalar entries here?

Reply:
+1, could we always use list?

Reply:
Sure! I can change it to list for all cases.
I was trying to make the basic usage simpler since I anticipate for most cases there will be one value.